profiler/src/CMakeLists.txt

```diff
@@ -57,11 +57,11 @@ ign_add_component(profiler SOURCES ${PROFILER_SRCS} GET_TARGET_NAME profiler_tar
 
 # Always enable profiler so that it's built, but make it
 # private so that downstream users can still disable profiling
-target_compile_definitions(${profiler_target} PRIVATE "IGN_PROFILER_ENABLE=1")
+target_compile_definitions(${profiler_target} PRIVATE "GZ_PROFILER_ENABLE=1")
```

Comment:
This compiler definition too, downstream projects use it, so we need to tick-tock

Reply:
https://github.com/gazebosim/gz-common/pull/366/commits/dcb9ca9654e4aeae7e60bcf21426085ff444f888